demos/src/main.scss

```diff
@@ -13,3 +13,7 @@ $o-card-is-silent: false;
 .demo-container--landscape {
 	width: 350px;
 }
+
+.o-card__tag__hover {
```

Comment:
Might be worth using a demo prefixed classname or similar to prevent confusing it with `o-card` styles.